{{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/model/model.py

```diff
@@ -27,9 +27,13 @@ def __init__(self):
         )
 
     def forward(self, prepared):
-        return architecture.PredictionBatch(logits=self.logits(prepared))
+        return model.PredictionBatch(logits=self.logits(prepared))
```

Comment:
`self.PredictionBatch`? Or is that bad/confusing?

Reply:
It's maybe a little more confusing

Reply:
I thought it could be less confusing since it's used under the `Model` namespace everywhere else, except where it's defined. But it doesn't really matter much.